neo-gui/update.xml

```diff
@@ -0,0 +1 @@
+<update minimum="2.9.0.0"></update>
```

Comment:
What happens if an older version reads this file?

Reply:
It show a dialog for try to update

Reply:
```suggestion
<?xml version="1.0" encoding="utf-8"?>
<update minimum="2.9.0.0"></update>
```